docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml

```diff
@@ -62,11 +76,53 @@ code: |
 include:
   - juvenile-sealing-10.yml
 ---
+comment: |
+  if click on docket sheets link
+    take to docket sheets flow
+  if click others
+    if haven't seen docket sheets link
+      take to docket sheets flow
+    else
+      go to next question...?
+
+  Clicking link triggers
+    new code block that
+      undefines values
+      triggers `url_action()`
+---
+id: tangent_start_state
+code: |
+  invalidate( 'wants_docket_instructions', 'stop_for_docket_sheets' )
+  which_tangent = None
+---
+id: tangent
+event: tangent
+code: |
+  which_tangent = action_argument('which_tangent')
+  if ( which_tangent == 'docket_sheets' ):
+    invalidate( 'wants_docket_instructions', 'stop_for_docket_sheets' )
+---
+id: docket_sheet_flow
+event: show_docket_sheet_tangent
+code: |
+  # How to hanlde progress in here? It's set to 1 when
+  # this is triggered. We can just put this logic
+  # before progress is started, but is that repeatable
+  # elsewhere? `progress: None` in questions? Justification:
+  # the user isn't making progress here. Discuss.
+  if ( wants_docket_instructions ):
```

Comment:
not 100% sure, but this seems like it could be condensed to 
```
if ( wants_docket_instructions and stop_for_docket_sheets ):
    not_finishing_end
invalidate( 'wants_docket_instructions', 'stop_for_docket_sheets' )
which_tangent = None
```

Reply:
I think we still need the `else` or we'll create a loop:
```
if ( wants_docket_instructions and stop_for_docket_sheets ):
    not_finishing_end
else:
    invalidate( 'wants_docket_instructions', 'stop_for_docket_sheets' )
    which_tangent = None
```

Reply:
Nope, that can't work. I take it back.

Reply:
Maybe it's just that I feel that the flow of including `else` is more explicit.

Reply:
Including the `else` would be fine I think, but I don't think we need 2 else's with the same content like we have now